subsys/dap/cmsis_dap.c

```diff
@@ -0,0 +1,1062 @@
+/*
+ * Copyright (c) 2018-2019 PHYTEC Messtechnik GmbH
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This file is based on DAP.c from DAPLink Interface Firmware.
+ * Copyright (c) 2009-2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
It might be nice to reference the commit SHA that this was started from. This way we can more easily compare if updates are made to the original ARM file.
Also, put a link to the original repo.

Reply:
Checked it against my presentation and updated accordingly, added version and link, no SHA.